pom.xml

```diff
@@ -186,6 +186,7 @@
         <maven-checkstyle-plugin.version>3.0.0</maven-checkstyle-plugin.version>
         <jacoco-maven-plugin.version>0.8.3</jacoco-maven-plugin.version>
         <jmh.version>1.21</jmh.version>
+        <nacos.version>1.0.0</nacos.version>
```

Comment:
Nacos version now shared in cluster management and DCS, you should use dependency management in the root of `aop-server/pom.xml`. The root pom of the project is used by both agent and oap.